src/content/learning/using-durable-objects.md

```diff
@@ -0,0 +1,216 @@
+---
+order: 8
+---
+
+# Using Durable Objects
+
+Durable Objects provide low-latency coordination and consistent storage for the Workers platform through two features: global uniqueness and a transactional storage API.
+
+* Global Uniqueness guarantees that there will be a single Durable Object with a given id running at once.  Requests for a Durable Object id are routed by the Workers runtime to the Cloudflare point-of-presence that owns the Durable Object.  
+
+* The transactional storage API provides strongly-consistent, key-value storage to the Durable Object.  Each Object can only read and modify keys associated with that Object. Execution of a Durable Object is single-threaded, but multiple request threads may be processed out-of-order from how they arrived at the Object.
```

Comment:
> but multiple request threads may be processed out-of-order from how they arrived at the Object.

The wording is here is a little strange. Maybe switch it to "multiple request events" instead of threads